src/aggregate.js

```diff
@@ -66,7 +67,15 @@ module.exports = async function aggregate(collection, params) {
    *
    * See mongo documentation: https://docs.mongodb.com/manual/reference/collation/#collation-and-index-use
    */
-  const options = config.COLLATION ? { collation: config.COLLATION } : undefined;
+  let options = config.COLLATION ? { collation: config.COLLATION } : {};
+
+  // Spread options passed in as a parameter
+  if (params.options) {
+    options = {
+      ...options,
+      ...params.options,
+    };
+  }
```

Comment:
**Comment**: If you've tested and it works, awesome. But I am always hesitant to use fancy-ish node ops like object spreading in potentially un-compiled node modules. Might have to tweak our babel config to make sure this is compiled if it doesn't work.

Reply:
confirmed, I have tested this.